
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<meta content="LinkML schemas for JGI and KBase genomics lakehouses" name="description"/>
<link href="../../../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.6.1, mkdocs-material-9.7.1" name="generator"/>
<title>Class: GtdbSpeciesClade - Bridge Schemas</title>
<link href="../../../assets/stylesheets/main.484c7ddc.min.css" rel="stylesheet"/>
<link href="../../../assets/stylesheets/palette.ab4e12ef.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("../../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body data-md-color-accent="cyan" data-md-color-primary="teal" data-md-color-scheme="default" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-gtdbspeciesclade">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header md-header--shadow" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="Bridge Schemas" class="md-header__button md-logo" data-md-component="logo" href="../../.." title="Bridge Schemas">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            Bridge Schemas
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: GtdbSpeciesClade
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="" tabindex="0">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list" role="presentation"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/cmungall/bridge-schemas" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="Bridge Schemas" class="md-nav__button md-logo" data-md-component="logo" href="../../.." title="Bridge Schemas">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"></path></svg>
</a>
    Bridge Schemas
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/cmungall/bridge-schemas" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../../..">
<span class="md-ellipsis">
    
  
    Home
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../methods/">
<span class="md-ellipsis">
    
  
    Methods
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../cross-database-joins/">
<span class="md-ellipsis">
    
  
    Cross-Database Joins
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../linkages/">
<span class="md-ellipsis">
    
  
    Database Linkages
  

    
  </span>
</a>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" id="__nav_5" type="checkbox"/>
<label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
<span class="md-ellipsis">
    
  
    Explanations
  

    
  </span>
<span class="md-nav__icon md-icon"></span>
</label>
<nav aria-expanded="false" aria-labelledby="__nav_5_label" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_5">
<span class="md-nav__icon md-icon"></span>
            
  
    Explanations
  

          </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../../../explanation/existing-schemas/">
<span class="md-ellipsis">
    
  
    Existing Schemas
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../explanation/traits/">
<span class="md-ellipsis">
    
  
    Microbial Traits
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../explanation/functional-annotation/">
<span class="md-ellipsis">
    
  
    Functional Annotation
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../explanation/taxonomic-coverage/">
<span class="md-ellipsis">
    
  
    Taxonomic Coverage
  

    
  </span>
</a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" id="__nav_6" type="checkbox"/>
<label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
<span class="md-ellipsis">
    
  
    Case Studies
  

    
  </span>
<span class="md-nav__icon md-icon"></span>
</label>
<nav aria-expanded="false" aria-labelledby="__nav_6_label" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_6">
<span class="md-nav__icon md-icon"></span>
            
  
    Case Studies
  

          </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../../../case-studies/">
<span class="md-ellipsis">
    
  
    Overview
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../case-studies/mll-bgc-methylorubrum/">
<span class="md-ellipsis">
    
  
    mll BGC (M. extorquens)
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../case-studies/microbial-isolates/">
<span class="md-ellipsis">
    
  
    Microbial Isolates
  

    
  </span>
</a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" id="__nav_7" type="checkbox"/>
<label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="0">
<span class="md-ellipsis">
    
  
    KBase/BERDL Schemas
  

    
  </span>
<span class="md-nav__icon md-icon"></span>
</label>
<nav aria-expanded="false" aria-labelledby="__nav_7_label" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_7">
<span class="md-nav__icon md-icon"></span>
            
  
    KBase/BERDL Schemas
  

          </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../">
<span class="md-ellipsis">
    
  
    Pangenome
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../nmdc_core/">
<span class="md-ellipsis">
    
  
    NMDC Core
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../kbase_genomes/">
<span class="md-ellipsis">
    
  
    Genomes
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../gapmind_pathways/">
<span class="md-ellipsis">
    
  
    GapMind Pathways
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../enigma_coral/">
<span class="md-ellipsis">
    
  
    ENIGMA CoRAL
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../phagefoundry_genome_browser/">
<span class="md-ellipsis">
    
  
    PhageFoundry Browser
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../phagefoundry_strain_modelling/">
<span class="md-ellipsis">
    
  
    PhageFoundry Modelling
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../kbase_uniprot_archaea/">
<span class="md-ellipsis">
    
  
    UniProt Archaea
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../kbase_uniprot_bacteria/">
<span class="md-ellipsis">
    
  
    UniProt Bacteria
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../kbase_uniref50/">
<span class="md-ellipsis">
    
  
    UniRef50
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../kbase_uniref90/">
<span class="md-ellipsis">
    
  
    UniRef90
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../kbase_uniref100/">
<span class="md-ellipsis">
    
  
    UniRef100
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../kbase_phenotype/">
<span class="md-ellipsis">
    
  
    Phenotype
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../kbase_msd_biochemistry/">
<span class="md-ellipsis">
    
  
    MSD Biochemistry
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../kbase_ontology_source/">
<span class="md-ellipsis">
    
  
    Ontology Source
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../kbase_refseq_taxon_api/">
<span class="md-ellipsis">
    
  
    RefSeq Taxon
  

    
  </span>
</a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item md-nav__item--nested">
<input class="md-nav__toggle md-toggle" id="__nav_8" type="checkbox"/>
<label class="md-nav__link" for="__nav_8" id="__nav_8_label" tabindex="0">
<span class="md-ellipsis">
    
  
    JGI Lakehouse Schemas
  

    
  </span>
<span class="md-nav__icon md-icon"></span>
</label>
<nav aria-expanded="false" aria-labelledby="__nav_8_label" class="md-nav" data-md-level="1">
<label class="md-nav__title" for="__nav_8">
<span class="md-nav__icon md-icon"></span>
            
  
    JGI Lakehouse Schemas
  

          </label>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/gold/">
<span class="md-ellipsis">
    
  
    GOLD
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_core_v400/">
<span class="md-ellipsis">
    
  
    IMG Core
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_ext/">
<span class="md-ellipsis">
    
  
    IMG Extended
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_gold/">
<span class="md-ellipsis">
    
  
    IMG GOLD
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_sat_v450/">
<span class="md-ellipsis">
    
  
    IMG Satellite
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_sub/">
<span class="md-ellipsis">
    
  
    IMG Submission
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/imgsg_dev/">
<span class="md-ellipsis">
    
  
    IMG Development
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_i_taxon/">
<span class="md-ellipsis">
    
  
    IMG Taxonomy
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_methylome/">
<span class="md-ellipsis">
    
  
    IMG Methylome
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_proteome/">
<span class="md-ellipsis">
    
  
    IMG Proteome
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_rnaseq/">
<span class="md-ellipsis">
    
  
    IMG RNAseq
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_mysql_abc/">
<span class="md-ellipsis">
    
  
    IMG MySQL ABC
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_mysql_img/">
<span class="md-ellipsis">
    
  
    IMG MySQL Core
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_mysql_imgvr_prod/">
<span class="md-ellipsis">
    
  
    IMG/VR
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_mysql_mbin/">
<span class="md-ellipsis">
    
  
    IMG MySQL MBin
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/img_mysql_misi/">
<span class="md-ellipsis">
    
  
    IMG MySQL MISI
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/phytozome/">
<span class="md-ellipsis">
    
  
    Phytozome
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/mycocosm/">
<span class="md-ellipsis">
    
  
    MycoCosm
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/gcs_citation/">
<span class="md-ellipsis">
    
  
    Citation Service
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/jgi_portal/">
<span class="md-ellipsis">
    
  
    JGI Portal
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/sdm_metadata/">
<span class="md-ellipsis">
    
  
    SDM Metadata
  

    
  </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../../../jgi/smc/">
<span class="md-ellipsis">
    
  
    SMC
  

    
  </span>
</a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
<span class="md-ellipsis">
      
        Slots
      
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#usages">
<span class="md-ellipsis">
      
        Usages
      
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
<span class="md-ellipsis">
      
        Identifier and Mapping Information
      
    </span>
</a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#annotations">
<span class="md-ellipsis">
      
        Annotations
      
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
<span class="md-ellipsis">
      
        Schema Source
      
    </span>
</a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
<span class="md-ellipsis">
      
        Mappings
      
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
<span class="md-ellipsis">
      
        LinkML Source
      
    </span>
</a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
<span class="md-ellipsis">
      
        Direct
      
    </span>
</a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
<span class="md-ellipsis">
      
        Induced
      
    </span>
</a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<h1 id="class-gtdbspeciesclade">Class: GtdbSpeciesClade</h1>
<p><em>GTDB species-level grouping with representative genome. Each clade represents a species cluster defined by 95% ANI threshold following GTDB taxonomy.</em></p>
<p><em>A species clade contains all genomes that cluster together at &gt;95% ANI with the representative genome. The representative is typically the highest-quality or type strain genome.</em></p>
<p><em>EXAMPLE CLADES (by genome count): - s__Staphylococcus_aureus--RS_GCF_001027105.1: 14,526 genomes - s__Klebsiella_pneumoniae--RS_GCF_000742135.1: 14,240 genomes - s__Salmonella_enterica--RS_GCF_000006945.2: 11,402 genomes</em></p>
<p><em>USAGE: Start here to explore pangenomes. Join to Pangenome for statistics, to Genome for individual assemblies, to GeneCluster for gene families.</em></p>
<p>URI: <a href="https://w3id.org/kbase/kbase_ke_pangenome/GtdbSpeciesClade">https://w3id.org/kbase/kbase_ke_pangenome/GtdbSpeciesClade</a></p>
<div class="mermaid"> classDiagram
    class GtdbSpeciesClade
    click GtdbSpeciesClade href "../GtdbSpeciesClade/"
      GtdbSpeciesClade : ANI_circumscription_radius

      GtdbSpeciesClade : GTDB_species

      GtdbSpeciesClade : gtdb_species_clade_id

      GtdbSpeciesClade : GTDB_taxonomy

      GtdbSpeciesClade : mean_intra_species_AF

      GtdbSpeciesClade : mean_intra_species_ANI

      GtdbSpeciesClade : min_intra_species_AF

      GtdbSpeciesClade : min_intra_species_ANI

      GtdbSpeciesClade : no_clustered_genomes_filtered

      GtdbSpeciesClade : no_clustered_genomes_unfiltered

      GtdbSpeciesClade : representative_genome_id





        GtdbSpeciesClade --&gt; "0..1" Genome : representative_genome_id
        click Genome href "../Genome/"




</div>
<!-- no inheritance hierarchy -->
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../gtdb_species_clade_id/">gtdb_species_clade_id</a></td>
<td>1 <br/> <a href="../String/">String</a></td>
<td>Species clade ID combining species name and representative genome</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../representative_genome_id/">representative_genome_id</a></td>
<td>0..1 <br/> <a href="../Genome/">Genome</a></td>
<td>Reference genome for this species</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../GTDB_species/">GTDB_species</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>GTDB species name with s__ prefix</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../GTDB_taxonomy/">GTDB_taxonomy</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Full GTDB lineage from domain to genus (species not repeated)</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../ANI_circumscription_radius/">ANI_circumscription_radius</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>ANI threshold for species membership</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../mean_intra_species_ANI/">mean_intra_species_ANI</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>Mean pairwise ANI among all genomes</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../min_intra_species_ANI/">min_intra_species_ANI</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>Minimum pairwise ANI observed</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../mean_intra_species_AF/">mean_intra_species_AF</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>Mean alignment fraction - proportion of genome aligning in ANI calculations</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../min_intra_species_AF/">min_intra_species_AF</a></td>
<td>0..1 <br/> <a href="../Float/">Float</a></td>
<td>Minimum alignment fraction observed between any two genomes</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../no_clustered_genomes_unfiltered/">no_clustered_genomes_unfiltered</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Total genomes assigned to species before quality filtering</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../no_clustered_genomes_filtered/">no_clustered_genomes_filtered</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>Genomes passing quality filters used in pangenome analysis</td>
<td>direct</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<table>
<thead>
<tr>
<th>used by</th>
<th>used in</th>
<th>type</th>
<th>used</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../Genome/">Genome</a></td>
<td><a href="../gtdb_species_clade_id/">gtdb_species_clade_id</a></td>
<td>range</td>
<td><a href="./">GtdbSpeciesClade</a></td>
</tr>
<tr>
<td><a href="../GeneCluster/">GeneCluster</a></td>
<td><a href="../gtdb_species_clade_id/">gtdb_species_clade_id</a></td>
<td>range</td>
<td><a href="./">GtdbSpeciesClade</a></td>
</tr>
<tr>
<td><a href="../Pangenome/">Pangenome</a></td>
<td><a href="../gtdb_species_clade_id/">gtdb_species_clade_id</a></td>
<td>range</td>
<td><a href="./">GtdbSpeciesClade</a></td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="annotations">Annotations</h3>
<table>
<thead>
<tr>
<th>property</th>
<th>value</th>
</tr>
</thead>
<tbody>
<tr>
<td>source_table</td>
<td>gtdb_species_clade</td>
</tr>
</tbody>
</table>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: https://w3id.org/kbase/kbase_ke_pangenome</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>https://w3id.org/kbase/kbase_ke_pangenome/GtdbSpeciesClade</td>
</tr>
<tr>
<td>native</td>
<td>https://w3id.org/kbase/kbase_ke_pangenome/GtdbSpeciesClade</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<div class="highlight"><pre><span></span><code><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="nt">annotations</span><span class="p">:</span>
<span class="w">  </span><span class="nt">source_table</span><span class="p">:</span>
<span class="w">    </span><span class="nt">tag</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">source_table</span>
<span class="w">    </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">gtdb_species_clade</span>
<span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="s">'GTDB</span><span class="nv"> </span><span class="s">species-level</span><span class="nv"> </span><span class="s">grouping</span><span class="nv"> </span><span class="s">with</span><span class="nv"> </span><span class="s">representative</span><span class="nv"> </span><span class="s">genome.</span><span class="nv"> </span><span class="s">Each</span><span class="nv"> </span><span class="s">clade</span><span class="nv"> </span><span class="s">represents</span>
<span class="w">  </span><span class="s">a</span><span class="nv"> </span><span class="s">species</span><span class="nv"> </span><span class="s">cluster</span><span class="nv"> </span><span class="s">defined</span><span class="nv"> </span><span class="s">by</span><span class="nv"> </span><span class="s">95%</span><span class="nv"> </span><span class="s">ANI</span><span class="nv"> </span><span class="s">threshold</span><span class="nv"> </span><span class="s">following</span><span class="nv"> </span><span class="s">GTDB</span><span class="nv"> </span><span class="s">taxonomy.</span>

<span class="w">  </span><span class="s">A</span><span class="nv"> </span><span class="s">species</span><span class="nv"> </span><span class="s">clade</span><span class="nv"> </span><span class="s">contains</span><span class="nv"> </span><span class="s">all</span><span class="nv"> </span><span class="s">genomes</span><span class="nv"> </span><span class="s">that</span><span class="nv"> </span><span class="s">cluster</span><span class="nv"> </span><span class="s">together</span><span class="nv"> </span><span class="s">at</span><span class="nv"> </span><span class="s">&gt;95%</span><span class="nv"> </span><span class="s">ANI</span><span class="nv"> </span><span class="s">with</span><span class="nv"> </span><span class="s">the</span>
<span class="w">  </span><span class="s">representative</span><span class="nv"> </span><span class="s">genome.</span><span class="nv"> </span><span class="s">The</span><span class="nv"> </span><span class="s">representative</span><span class="nv"> </span><span class="s">is</span><span class="nv"> </span><span class="s">typically</span><span class="nv"> </span><span class="s">the</span><span class="nv"> </span><span class="s">highest-quality</span><span class="nv"> </span><span class="s">or</span><span class="nv"> </span><span class="s">type</span>
<span class="w">  </span><span class="s">strain</span><span class="nv"> </span><span class="s">genome.</span>

<span class="w">  </span><span class="s">EXAMPLE</span><span class="nv"> </span><span class="s">CLADES</span><span class="nv"> </span><span class="s">(by</span><span class="nv"> </span><span class="s">genome</span><span class="nv"> </span><span class="s">count):</span><span class="nv"> </span><span class="s">-</span><span class="nv"> </span><span class="s">s__Staphylococcus_aureus--RS_GCF_001027105.1:</span>
<span class="w">  </span><span class="s">14,526</span><span class="nv"> </span><span class="s">genomes</span><span class="nv"> </span><span class="s">-</span><span class="nv"> </span><span class="s">s__Klebsiella_pneumoniae--RS_GCF_000742135.1:</span><span class="nv"> </span><span class="s">14,240</span><span class="nv"> </span><span class="s">genomes</span><span class="nv"> </span><span class="s">-</span>
<span class="w">  </span><span class="s">s__Salmonella_enterica--RS_GCF_000006945.2:</span><span class="nv"> </span><span class="s">11,402</span><span class="nv"> </span><span class="s">genomes</span>

<span class="w">  </span><span class="s">USAGE:</span><span class="nv"> </span><span class="s">Start</span><span class="nv"> </span><span class="s">here</span><span class="nv"> </span><span class="s">to</span><span class="nv"> </span><span class="s">explore</span><span class="nv"> </span><span class="s">pangenomes.</span><span class="nv"> </span><span class="s">Join</span><span class="nv"> </span><span class="s">to</span><span class="nv"> </span><span class="s">Pangenome</span><span class="nv"> </span><span class="s">for</span><span class="nv"> </span><span class="s">statistics,</span><span class="nv"> </span><span class="s">to</span><span class="nv"> </span><span class="s">Genome</span>
<span class="w">  </span><span class="s">for</span><span class="nv"> </span><span class="s">individual</span><span class="nv"> </span><span class="s">assemblies,</span><span class="nv"> </span><span class="s">to</span><span class="nv"> </span><span class="s">GeneCluster</span><span class="nv"> </span><span class="s">for</span><span class="nv"> </span><span class="s">gene</span><span class="nv"> </span><span class="s">families.'</span>
<span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="nt">attributes</span><span class="p">:</span>
<span class="w">  </span><span class="nt">gtdb_species_clade_id</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">gtdb_species_clade_id</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="s">'Species</span><span class="nv"> </span><span class="s">clade</span><span class="nv"> </span><span class="s">ID</span><span class="nv"> </span><span class="s">combining</span><span class="nv"> </span><span class="s">species</span><span class="nv"> </span><span class="s">name</span><span class="nv"> </span><span class="s">and</span><span class="nv"> </span><span class="s">representative</span><span class="nv"> </span><span class="s">genome.</span>
<span class="w">      </span><span class="s">Format:</span><span class="nv"> </span><span class="s">s__Genus_species--{RS|GB}_GC{F|A}_XXXXXXXXX.X</span>

<span class="w">      </span><span class="s">The</span><span class="nv"> </span><span class="s">ID</span><span class="nv"> </span><span class="s">encodes:</span><span class="nv"> </span><span class="s">species</span><span class="nv"> </span><span class="s">name</span><span class="nv"> </span><span class="s">(s__prefix),</span><span class="nv"> </span><span class="s">representative</span><span class="nv"> </span><span class="s">source</span><span class="nv"> </span><span class="s">(RS=RefSeq,</span>
<span class="w">      </span><span class="s">GB=GenBank),</span><span class="nv"> </span><span class="s">and</span><span class="nv"> </span><span class="s">assembly</span><span class="nv"> </span><span class="s">accession.'</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Klebsiella_pneumoniae--RS_GCF_000742135.1</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">K. pneumoniae - major human pathogen, 14K+ genomes</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Staphylococcus_aureus--RS_GCF_001027105.1</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">S. aureus - most genomes in database</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Escherichia_coli--RS_GCF_000005845.2</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">E. coli K-12 - model organism</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Mycobacterium_tuberculosis--RS_GCF_000195955.2</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">TB pathogen - highly clonal species</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Pseudomonas_aeruginosa--RS_GCF_001457615.1</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Opportunistic pathogen with large pangenome</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">identifier</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">true</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Genome</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GeneCluster</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Pangenome</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">string</span>
<span class="w">    </span><span class="nt">required</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">true</span>
<span class="w">    </span><span class="nt">pattern</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__[A-Za-z0-9_]+--[A-Z]{2}_GC[AF]_\d+\.\d+</span>
<span class="w">  </span><span class="nt">representative_genome_id</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">representative_genome_id</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Reference genome for this species. Typically highest quality assembly</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">or type strain. Used as anchor for ANI calculations.</span>
<span class="w">    </span><span class="nt">comments</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="s">'Foreign</span><span class="nv"> </span><span class="s">key:</span><span class="nv"> </span><span class="s">Genome.genome_id'</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">RS_GCF_000742135.1</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">RS_GCF_001027105.1</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Genome</span>
<span class="w">  </span><span class="nt">GTDB_species</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GTDB_species</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GTDB species name with s__ prefix. May differ from NCBI species name</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">due to GTDB's genome-based taxonomy.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Klebsiella_pneumoniae</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Staphylococcus_aureus</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Escherichia_coli</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Bacillus_subtilis</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">string</span>
<span class="w">    </span><span class="nt">pattern</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__[A-Za-z0-9_]+</span>
<span class="w">  </span><span class="nt">GTDB_taxonomy</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GTDB_taxonomy</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="s">'Full</span><span class="nv"> </span><span class="s">GTDB</span><span class="nv"> </span><span class="s">lineage</span><span class="nv"> </span><span class="s">from</span><span class="nv"> </span><span class="s">domain</span><span class="nv"> </span><span class="s">to</span><span class="nv"> </span><span class="s">genus</span><span class="nv"> </span><span class="s">(species</span><span class="nv"> </span><span class="s">not</span><span class="nv"> </span><span class="s">repeated).</span><span class="nv"> </span><span class="s">Semicolon-separated</span>
<span class="w">      </span><span class="s">with</span><span class="nv"> </span><span class="s">rank</span><span class="nv"> </span><span class="s">prefixes:</span><span class="nv"> </span><span class="s">d__,</span><span class="nv"> </span><span class="s">p__,</span><span class="nv"> </span><span class="s">c__,</span><span class="nv"> </span><span class="s">o__,</span><span class="nv"> </span><span class="s">f__,</span><span class="nv"> </span><span class="s">g__'</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">d__Bacteria;p__Pseudomonadota;c__Gammaproteobacteria;o__Enterobacterales;f__Enterobacteriaceae;g__Klebsiella</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">d__Bacteria;p__Bacillota;c__Bacilli;o__Staphylococcales;f__Staphylococcaceae;g__Staphylococcus</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">d__Archaea;p__Methanobacteriota_B;c__Thermococci;o__Thermococcales;f__Thermococcaceae;g__Thermococcus_A</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">string</span>
<span class="w">  </span><span class="nt">ANI_circumscription_radius</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">ANI_circumscription_radius</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">ANI threshold for species membership. Typically 95% for most species.</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">Some species have tighter boundaries (higher values).</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'95.0'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Standard species threshold</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'95.239'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">K. pneumoniae threshold</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'97.08'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Tighter threshold for clonal species</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">float</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">90.0</span>
<span class="w">    </span><span class="nt">maximum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">100.0</span>
<span class="w">    </span><span class="nt">unit</span><span class="p">:</span>
<span class="w">      </span><span class="nt">ucum_code</span><span class="p">:</span><span class="w"> </span><span class="s">'%'</span>
<span class="w">  </span><span class="nt">mean_intra_species_ANI</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">mean_intra_species_ANI</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Mean pairwise ANI among all genomes. Higher values indicate more</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">clonal/homogeneous species. Lower values suggest higher diversity.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'98.97'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">K. pneumoniae - moderately diverse</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'99.5'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Highly clonal species (e.g., M. tuberculosis)</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'96.5'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Highly diverse species</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">float</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">95.0</span>
<span class="w">    </span><span class="nt">maximum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">100.0</span>
<span class="w">  </span><span class="nt">min_intra_species_ANI</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">min_intra_species_ANI</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Minimum pairwise ANI observed. Low values near 95% indicate species</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">at boundary of splitting into subspecies.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'95.28'</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'97.08'</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">float</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">90.0</span>
<span class="w">    </span><span class="nt">maximum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">100.0</span>
<span class="w">  </span><span class="nt">mean_intra_species_AF</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">mean_intra_species_AF</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Mean alignment fraction - proportion of genome aligning in ANI calculations.</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">Low AF may indicate accessory genome differences.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'0.88'</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'0.95'</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">float</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">0.0</span>
<span class="w">    </span><span class="nt">maximum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1.0</span>
<span class="w">  </span><span class="nt">min_intra_species_AF</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">min_intra_species_AF</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Minimum alignment fraction observed between any two genomes</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'0.76'</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'0.82'</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">float</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">0.0</span>
<span class="w">    </span><span class="nt">maximum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1.0</span>
<span class="w">  </span><span class="nt">no_clustered_genomes_unfiltered</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">no_clustered_genomes_unfiltered</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Total genomes assigned to species before quality filtering. Difference</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">from filtered count indicates low-quality genomes removed.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'14975'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">K. pneumoniae unfiltered</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'14959'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">S. aureus unfiltered</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">integer</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1</span>
<span class="w">  </span><span class="nt">no_clustered_genomes_filtered</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">no_clustered_genomes_filtered</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Genomes passing quality filters used in pangenome analysis. These</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">genomes have sufficient completeness and low contamination.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'14240'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">K. pneumoniae after filtering</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'14526'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">S. aureus after filtering</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">integer</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1</span>
</code></pre></div>
</details>
<h3 id="induced">Induced</h3>
<details>
<div class="highlight"><pre><span></span><code><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="nt">annotations</span><span class="p">:</span>
<span class="w">  </span><span class="nt">source_table</span><span class="p">:</span>
<span class="w">    </span><span class="nt">tag</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">source_table</span>
<span class="w">    </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">gtdb_species_clade</span>
<span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="s">'GTDB</span><span class="nv"> </span><span class="s">species-level</span><span class="nv"> </span><span class="s">grouping</span><span class="nv"> </span><span class="s">with</span><span class="nv"> </span><span class="s">representative</span><span class="nv"> </span><span class="s">genome.</span><span class="nv"> </span><span class="s">Each</span><span class="nv"> </span><span class="s">clade</span><span class="nv"> </span><span class="s">represents</span>
<span class="w">  </span><span class="s">a</span><span class="nv"> </span><span class="s">species</span><span class="nv"> </span><span class="s">cluster</span><span class="nv"> </span><span class="s">defined</span><span class="nv"> </span><span class="s">by</span><span class="nv"> </span><span class="s">95%</span><span class="nv"> </span><span class="s">ANI</span><span class="nv"> </span><span class="s">threshold</span><span class="nv"> </span><span class="s">following</span><span class="nv"> </span><span class="s">GTDB</span><span class="nv"> </span><span class="s">taxonomy.</span>

<span class="w">  </span><span class="s">A</span><span class="nv"> </span><span class="s">species</span><span class="nv"> </span><span class="s">clade</span><span class="nv"> </span><span class="s">contains</span><span class="nv"> </span><span class="s">all</span><span class="nv"> </span><span class="s">genomes</span><span class="nv"> </span><span class="s">that</span><span class="nv"> </span><span class="s">cluster</span><span class="nv"> </span><span class="s">together</span><span class="nv"> </span><span class="s">at</span><span class="nv"> </span><span class="s">&gt;95%</span><span class="nv"> </span><span class="s">ANI</span><span class="nv"> </span><span class="s">with</span><span class="nv"> </span><span class="s">the</span>
<span class="w">  </span><span class="s">representative</span><span class="nv"> </span><span class="s">genome.</span><span class="nv"> </span><span class="s">The</span><span class="nv"> </span><span class="s">representative</span><span class="nv"> </span><span class="s">is</span><span class="nv"> </span><span class="s">typically</span><span class="nv"> </span><span class="s">the</span><span class="nv"> </span><span class="s">highest-quality</span><span class="nv"> </span><span class="s">or</span><span class="nv"> </span><span class="s">type</span>
<span class="w">  </span><span class="s">strain</span><span class="nv"> </span><span class="s">genome.</span>

<span class="w">  </span><span class="s">EXAMPLE</span><span class="nv"> </span><span class="s">CLADES</span><span class="nv"> </span><span class="s">(by</span><span class="nv"> </span><span class="s">genome</span><span class="nv"> </span><span class="s">count):</span><span class="nv"> </span><span class="s">-</span><span class="nv"> </span><span class="s">s__Staphylococcus_aureus--RS_GCF_001027105.1:</span>
<span class="w">  </span><span class="s">14,526</span><span class="nv"> </span><span class="s">genomes</span><span class="nv"> </span><span class="s">-</span><span class="nv"> </span><span class="s">s__Klebsiella_pneumoniae--RS_GCF_000742135.1:</span><span class="nv"> </span><span class="s">14,240</span><span class="nv"> </span><span class="s">genomes</span><span class="nv"> </span><span class="s">-</span>
<span class="w">  </span><span class="s">s__Salmonella_enterica--RS_GCF_000006945.2:</span><span class="nv"> </span><span class="s">11,402</span><span class="nv"> </span><span class="s">genomes</span>

<span class="w">  </span><span class="s">USAGE:</span><span class="nv"> </span><span class="s">Start</span><span class="nv"> </span><span class="s">here</span><span class="nv"> </span><span class="s">to</span><span class="nv"> </span><span class="s">explore</span><span class="nv"> </span><span class="s">pangenomes.</span><span class="nv"> </span><span class="s">Join</span><span class="nv"> </span><span class="s">to</span><span class="nv"> </span><span class="s">Pangenome</span><span class="nv"> </span><span class="s">for</span><span class="nv"> </span><span class="s">statistics,</span><span class="nv"> </span><span class="s">to</span><span class="nv"> </span><span class="s">Genome</span>
<span class="w">  </span><span class="s">for</span><span class="nv"> </span><span class="s">individual</span><span class="nv"> </span><span class="s">assemblies,</span><span class="nv"> </span><span class="s">to</span><span class="nv"> </span><span class="s">GeneCluster</span><span class="nv"> </span><span class="s">for</span><span class="nv"> </span><span class="s">gene</span><span class="nv"> </span><span class="s">families.'</span>
<span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="nt">attributes</span><span class="p">:</span>
<span class="w">  </span><span class="nt">gtdb_species_clade_id</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">gtdb_species_clade_id</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="s">'Species</span><span class="nv"> </span><span class="s">clade</span><span class="nv"> </span><span class="s">ID</span><span class="nv"> </span><span class="s">combining</span><span class="nv"> </span><span class="s">species</span><span class="nv"> </span><span class="s">name</span><span class="nv"> </span><span class="s">and</span><span class="nv"> </span><span class="s">representative</span><span class="nv"> </span><span class="s">genome.</span>
<span class="w">      </span><span class="s">Format:</span><span class="nv"> </span><span class="s">s__Genus_species--{RS|GB}_GC{F|A}_XXXXXXXXX.X</span>

<span class="w">      </span><span class="s">The</span><span class="nv"> </span><span class="s">ID</span><span class="nv"> </span><span class="s">encodes:</span><span class="nv"> </span><span class="s">species</span><span class="nv"> </span><span class="s">name</span><span class="nv"> </span><span class="s">(s__prefix),</span><span class="nv"> </span><span class="s">representative</span><span class="nv"> </span><span class="s">source</span><span class="nv"> </span><span class="s">(RS=RefSeq,</span>
<span class="w">      </span><span class="s">GB=GenBank),</span><span class="nv"> </span><span class="s">and</span><span class="nv"> </span><span class="s">assembly</span><span class="nv"> </span><span class="s">accession.'</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Klebsiella_pneumoniae--RS_GCF_000742135.1</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">K. pneumoniae - major human pathogen, 14K+ genomes</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Staphylococcus_aureus--RS_GCF_001027105.1</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">S. aureus - most genomes in database</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Escherichia_coli--RS_GCF_000005845.2</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">E. coli K-12 - model organism</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Mycobacterium_tuberculosis--RS_GCF_000195955.2</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">TB pathogen - highly clonal species</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Pseudomonas_aeruginosa--RS_GCF_001457615.1</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Opportunistic pathogen with large pangenome</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">identifier</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">true</span>
<span class="w">    </span><span class="nt">alias</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">gtdb_species_clade_id</span>
<span class="w">    </span><span class="nt">owner</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Genome</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GeneCluster</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Pangenome</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">string</span>
<span class="w">    </span><span class="nt">required</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">true</span>
<span class="w">    </span><span class="nt">pattern</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__[A-Za-z0-9_]+--[A-Z]{2}_GC[AF]_\d+\.\d+</span>
<span class="w">  </span><span class="nt">representative_genome_id</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">representative_genome_id</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Reference genome for this species. Typically highest quality assembly</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">or type strain. Used as anchor for ANI calculations.</span>
<span class="w">    </span><span class="nt">comments</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="s">'Foreign</span><span class="nv"> </span><span class="s">key:</span><span class="nv"> </span><span class="s">Genome.genome_id'</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">RS_GCF_000742135.1</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">RS_GCF_001027105.1</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">alias</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">representative_genome_id</span>
<span class="w">    </span><span class="nt">owner</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Genome</span>
<span class="w">  </span><span class="nt">GTDB_species</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GTDB_species</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GTDB species name with s__ prefix. May differ from NCBI species name</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">due to GTDB's genome-based taxonomy.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Klebsiella_pneumoniae</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Staphylococcus_aureus</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Escherichia_coli</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__Bacillus_subtilis</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">alias</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GTDB_species</span>
<span class="w">    </span><span class="nt">owner</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">string</span>
<span class="w">    </span><span class="nt">pattern</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">s__[A-Za-z0-9_]+</span>
<span class="w">  </span><span class="nt">GTDB_taxonomy</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GTDB_taxonomy</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="s">'Full</span><span class="nv"> </span><span class="s">GTDB</span><span class="nv"> </span><span class="s">lineage</span><span class="nv"> </span><span class="s">from</span><span class="nv"> </span><span class="s">domain</span><span class="nv"> </span><span class="s">to</span><span class="nv"> </span><span class="s">genus</span><span class="nv"> </span><span class="s">(species</span><span class="nv"> </span><span class="s">not</span><span class="nv"> </span><span class="s">repeated).</span><span class="nv"> </span><span class="s">Semicolon-separated</span>
<span class="w">      </span><span class="s">with</span><span class="nv"> </span><span class="s">rank</span><span class="nv"> </span><span class="s">prefixes:</span><span class="nv"> </span><span class="s">d__,</span><span class="nv"> </span><span class="s">p__,</span><span class="nv"> </span><span class="s">c__,</span><span class="nv"> </span><span class="s">o__,</span><span class="nv"> </span><span class="s">f__,</span><span class="nv"> </span><span class="s">g__'</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">d__Bacteria;p__Pseudomonadota;c__Gammaproteobacteria;o__Enterobacterales;f__Enterobacteriaceae;g__Klebsiella</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">d__Bacteria;p__Bacillota;c__Bacilli;o__Staphylococcales;f__Staphylococcaceae;g__Staphylococcus</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">d__Archaea;p__Methanobacteriota_B;c__Thermococci;o__Thermococcales;f__Thermococcaceae;g__Thermococcus_A</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">alias</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GTDB_taxonomy</span>
<span class="w">    </span><span class="nt">owner</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">string</span>
<span class="w">  </span><span class="nt">ANI_circumscription_radius</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">ANI_circumscription_radius</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">ANI threshold for species membership. Typically 95% for most species.</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">Some species have tighter boundaries (higher values).</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'95.0'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Standard species threshold</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'95.239'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">K. pneumoniae threshold</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'97.08'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Tighter threshold for clonal species</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">alias</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">ANI_circumscription_radius</span>
<span class="w">    </span><span class="nt">owner</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">float</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">90.0</span>
<span class="w">    </span><span class="nt">maximum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">100.0</span>
<span class="w">    </span><span class="nt">unit</span><span class="p">:</span>
<span class="w">      </span><span class="nt">ucum_code</span><span class="p">:</span><span class="w"> </span><span class="s">'%'</span>
<span class="w">  </span><span class="nt">mean_intra_species_ANI</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">mean_intra_species_ANI</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Mean pairwise ANI among all genomes. Higher values indicate more</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">clonal/homogeneous species. Lower values suggest higher diversity.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'98.97'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">K. pneumoniae - moderately diverse</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'99.5'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Highly clonal species (e.g., M. tuberculosis)</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'96.5'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Highly diverse species</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">alias</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">mean_intra_species_ANI</span>
<span class="w">    </span><span class="nt">owner</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">float</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">95.0</span>
<span class="w">    </span><span class="nt">maximum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">100.0</span>
<span class="w">  </span><span class="nt">min_intra_species_ANI</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">min_intra_species_ANI</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Minimum pairwise ANI observed. Low values near 95% indicate species</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">at boundary of splitting into subspecies.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'95.28'</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'97.08'</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">alias</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">min_intra_species_ANI</span>
<span class="w">    </span><span class="nt">owner</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">float</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">90.0</span>
<span class="w">    </span><span class="nt">maximum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">100.0</span>
<span class="w">  </span><span class="nt">mean_intra_species_AF</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">mean_intra_species_AF</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Mean alignment fraction - proportion of genome aligning in ANI calculations.</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">Low AF may indicate accessory genome differences.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'0.88'</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'0.95'</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">alias</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">mean_intra_species_AF</span>
<span class="w">    </span><span class="nt">owner</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">float</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">0.0</span>
<span class="w">    </span><span class="nt">maximum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1.0</span>
<span class="w">  </span><span class="nt">min_intra_species_AF</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">min_intra_species_AF</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Minimum alignment fraction observed between any two genomes</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'0.76'</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'0.82'</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">alias</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">min_intra_species_AF</span>
<span class="w">    </span><span class="nt">owner</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">float</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">0.0</span>
<span class="w">    </span><span class="nt">maximum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1.0</span>
<span class="w">  </span><span class="nt">no_clustered_genomes_unfiltered</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">no_clustered_genomes_unfiltered</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Total genomes assigned to species before quality filtering. Difference</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">from filtered count indicates low-quality genomes removed.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'14975'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">K. pneumoniae unfiltered</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'14959'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">S. aureus unfiltered</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">alias</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">no_clustered_genomes_unfiltered</span>
<span class="w">    </span><span class="nt">owner</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">integer</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1</span>
<span class="w">  </span><span class="nt">no_clustered_genomes_filtered</span><span class="p">:</span>
<span class="w">    </span><span class="nt">name</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">no_clustered_genomes_filtered</span>
<span class="w">    </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">Genomes passing quality filters used in pangenome analysis. These</span>
<span class="w">      </span><span class="l l-Scalar l-Scalar-Plain">genomes have sufficient completeness and low contamination.</span>
<span class="w">    </span><span class="nt">examples</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'14240'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">K. pneumoniae after filtering</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="nt">value</span><span class="p">:</span><span class="w"> </span><span class="s">'14526'</span>
<span class="w">      </span><span class="nt">description</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">S. aureus after filtering</span>
<span class="w">    </span><span class="nt">from_schema</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">https://w3id.org/kbase/kbase_ke_pangenome</span>
<span class="w">    </span><span class="nt">rank</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1000</span>
<span class="w">    </span><span class="nt">alias</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">no_clustered_genomes_filtered</span>
<span class="w">    </span><span class="nt">owner</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">domain_of</span><span class="p">:</span>
<span class="w">    </span><span class="p p-Indicator">-</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">GtdbSpeciesClade</span>
<span class="w">    </span><span class="nt">range</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">integer</span>
<span class="w">    </span><span class="nt">minimum_value</span><span class="p">:</span><span class="w"> </span><span class="l l-Scalar l-Scalar-Plain">1</span>
</code></pre></div>
</details>
</article>
</div>
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"annotate": null, "base": "../../..", "features": [], "search": "../../../assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
<script src="../../../assets/javascripts/bundle.79ae519e.min.js"></script>
<script type="module">import mermaid from "https://unpkg.com/mermaid@10.4.0/dist/mermaid.esm.min.mjs";
mermaid.initialize({});</script></body>
</html>