
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="LinkML schemas for JGI and KBase genomics lakehouses">
      
      
      
      
        <link rel="prev" href="../kbase_ke_pangenome/">
      
      
        <link rel="next" href="../kbase_genomes/">
      
      
        
      
      
      <link rel="icon" href="../../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.1">
    
    
      
        <title>NMDC Core - Bridge Schemas</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.484c7ddc.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.ab4e12ef.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="teal" data-md-color-accent="cyan">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#nmdc-core-database" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Bridge Schemas" class="md-header__button md-logo" aria-label="Bridge Schemas" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Bridge Schemas
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              NMDC Core
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
      <div class="md-header__source">
        <a href="https://github.com/cmungall/bridge-schemas" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Bridge Schemas" class="md-nav__button md-logo" aria-label="Bridge Schemas" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    Bridge Schemas
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/cmungall/bridge-schemas" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 7.1.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2025 Fonticons, Inc.--><path d="M439.6 236.1 244 40.5c-5.4-5.5-12.8-8.5-20.4-8.5s-15 3-20.4 8.4L162.5 81l51.5 51.5c27.1-9.1 52.7 16.8 43.4 43.7l49.7 49.7c34.2-11.8 61.2 31 35.5 56.7-26.5 26.5-70.2-2.9-56-37.3L240.3 199v121.9c25.3 12.5 22.3 41.8 9.1 55-6.4 6.4-15.2 10.1-24.3 10.1s-17.8-3.6-24.3-10.1c-17.6-17.6-11.1-46.9 11.2-56v-123c-20.8-8.5-24.6-30.7-18.6-45L142.6 101 8.5 235.1C3 240.6 0 247.9 0 255.5s3 15 8.5 20.4l195.6 195.7c5.4 5.4 12.7 8.4 20.4 8.4s15-3 20.4-8.4l194.7-194.7c5.4-5.4 8.4-12.8 8.4-20.4s-3-15-8.4-20.4"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Home
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../methods/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Methods
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../cross-database-joins/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Cross-Database Joins
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../../linkages/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Database Linkages
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Explanations
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            
  
    Explanations
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../explanation/existing-schemas/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Existing Schemas
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../explanation/traits/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Microbial Traits
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../explanation/functional-annotation/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Functional Annotation
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../explanation/taxonomic-coverage/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Taxonomic Coverage
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_6" >
        
          
          <label class="md-nav__link" for="__nav_6" id="__nav_6_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Case Studies
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_6_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_6">
            <span class="md-nav__icon md-icon"></span>
            
  
    Case Studies
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../case-studies/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Overview
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../case-studies/mll-bgc-methylorubrum/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    mll BGC (M. extorquens)
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../case-studies/microbial-isolates/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Microbial Isolates
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_7" checked>
        
          
          <label class="md-nav__link" for="__nav_7" id="__nav_7_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    KBase/BERDL Schemas
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_7_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_7">
            <span class="md-nav__icon md-icon"></span>
            
  
    KBase/BERDL Schemas
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../kbase_ke_pangenome/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Pangenome
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    
  
    NMDC Core
  

    
  </span>
  
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    
  
    NMDC Core
  

    
  </span>
  
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      
        Classes
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      
        Slots
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      
        Enumerations
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      
        Types
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      
        Subsets
      
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../kbase_genomes/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Genomes
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../gapmind_pathways/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    GapMind Pathways
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../enigma_coral/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    ENIGMA CoRAL
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../phagefoundry_genome_browser/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    PhageFoundry Browser
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../phagefoundry_strain_modelling/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    PhageFoundry Modelling
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../kbase_uniprot_archaea/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    UniProt Archaea
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../kbase_uniprot_bacteria/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    UniProt Bacteria
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../kbase_uniref50/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    UniRef50
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../kbase_uniref90/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    UniRef90
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../kbase_uniref100/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    UniRef100
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../kbase_phenotype/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Phenotype
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../kbase_msd_biochemistry/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    MSD Biochemistry
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../kbase_ontology_source/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Ontology Source
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../kbase_refseq_taxon_api/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    RefSeq Taxon
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_8" >
        
          
          <label class="md-nav__link" for="__nav_8" id="__nav_8_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    JGI Lakehouse Schemas
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_8_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_8">
            <span class="md-nav__icon md-icon"></span>
            
  
    JGI Lakehouse Schemas
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/gold/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    GOLD
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_core_v400/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG Core
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_ext/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG Extended
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_gold/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG GOLD
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_sat_v450/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG Satellite
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_sub/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG Submission
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/imgsg_dev/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG Development
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_i_taxon/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG Taxonomy
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_methylome/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG Methylome
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_proteome/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG Proteome
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_rnaseq/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG RNAseq
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_mysql_abc/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG MySQL ABC
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_mysql_img/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG MySQL Core
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_mysql_imgvr_prod/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG/VR
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_mysql_mbin/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG MySQL MBin
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/img_mysql_misi/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    IMG MySQL MISI
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/phytozome/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Phytozome
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/mycocosm/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    MycoCosm
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/gcs_citation/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Citation Service
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/jgi_portal/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    JGI Portal
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/sdm_metadata/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    SDM Metadata
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../jgi/smc/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    SMC
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#classes" class="md-nav__link">
    <span class="md-ellipsis">
      
        Classes
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#slots" class="md-nav__link">
    <span class="md-ellipsis">
      
        Slots
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#enumerations" class="md-nav__link">
    <span class="md-ellipsis">
      
        Enumerations
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#types" class="md-nav__link">
    <span class="md-ellipsis">
      
        Types
      
    </span>
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#subsets" class="md-nav__link">
    <span class="md-ellipsis">
      
        Subsets
      
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                
                  


  
  


<h1 id="nmdc-core-database">NMDC Core Database</h1>
<p>National Microbiome Data Collaborative (NMDC) core data including samples, studies, omics data, annotations, and embeddings. Contains data from GOLD, metabolomics, metagenomics, and other microbiome sources. Supports multi-modal analysis through unified embeddings and tokenization.
DATABASE STATISTICS (as of 2024): - 48 research studies with comprehensive metadata - 3,129,061+ metabolomics feature records - 48,196 annotation terms (GO, EC, KEGG, COG, MetaCyc) - 39,354 non-obsolete GO terms - 256-dimensional unified embeddings per biosample
ANNOTATION TERM COUNTS BY SOURCE: | Source       | Terms  | Description                        | |--------------|--------|-------------------------------------| | GO           | 48,196 | Gene Ontology terms                | | EC           | 8,813  | Enzyme Commission numbers          | | KEGG KO      | 8,104  | KEGG Orthology functional orthologs| | MetaCyc      | 1,538  | MetaCyc metabolic pathways         | | KEGG Module  | 370    | KEGG functional modules            | | KEGG Pathway | 306    | KEGG metabolic pathways            | | COG          | 26     | Clusters of Orthologous Groups     |
GO TERM DISTRIBUTION BY NAMESPACE: | Namespace          | Count  | Fraction | |--------------------|--------|----------| | biological_process | 30,817 | 64%      | | molecular_function | 12,805 | 27%      | | cellular_component | 4,573  | 9%       |
ECOSYSTEM COVERAGE (studies with ecosystem data): - Environmental/Terrestrial/Soil (3 studies) - Environmental/Aquatic/Freshwater (3 studies) - Environmental/Terrestrial/Deep subsurface (1 study) - Host-associated/Plants (1 study)
KEY FEATURES: - Multi-modal embeddings for similarity search across samples - Unified annotation vocabulary across GO, KEGG, EC, COG, MetaCyc - Pre-computed GO hierarchy for efficient ancestor queries - Mass spectrometry metabolomics with compound identification - Taxonomic classification from multiple tools (Kraken, GOTTCHA, Centrifuge)
USAGE: For functional analysis, start with annotation_terms_unified. For metabolomics, use metabolomics_gold. For similarity search, use embeddings_v1. For GO-based enrichment, use go_hierarchy_flat for closure queries.</p>
<p>URI: https://w3id.org/kbase/nmdc_core</p>
<p>Name: nmdc_core</p>
<h2 id="classes">Classes</h2>
<table>
<thead>
<tr>
<th>Class</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="AbioticEmbeddings/">AbioticEmbeddings</a></td>
<td>Abiotic factor embeddings from environmental measurements (pH, temperature, m...</td>
</tr>
<tr>
<td><a href="AbioticFeatures/">AbioticFeatures</a></td>
<td>Abiotic environmental features for machine learning</td>
</tr>
<tr>
<td><a href="AnnotationCrossrefs/">AnnotationCrossrefs</a></td>
<td>Cross-references between annotation databases</td>
</tr>
<tr>
<td><a href="AnnotationHierarchiesUnified/">AnnotationHierarchiesUnified</a></td>
<td>Unified annotation hierarchies across sources</td>
</tr>
<tr>
<td><a href="AnnotationTermsUnified/">AnnotationTermsUnified</a></td>
<td>Unified annotation terms across sources (GO, KEGG, EC, COG, MetaCyc)</td>
</tr>
<tr>
<td><a href="BiochemicalEmbeddings/">BiochemicalEmbeddings</a></td>
<td>Biochemical feature embeddings from metabolomics data</td>
</tr>
<tr>
<td><a href="BiochemicalFeatures/">BiochemicalFeatures</a></td>
<td>Biochemical features from metabolomics for machine learning</td>
</tr>
<tr>
<td><a href="CentrifugeGold/">CentrifugeGold</a></td>
<td>Centrifuge taxonomic classifications for GOLD samples</td>
</tr>
<tr>
<td><a href="CogCategories/">CogCategories</a></td>
<td>COG functional categories with descriptions and colors</td>
</tr>
<tr>
<td><a href="ContigTaxonomy/">ContigTaxonomy</a></td>
<td>Contig-level taxonomic assignments from metagenome assemblies</td>
</tr>
<tr>
<td><a href="EcTerms/">EcTerms</a></td>
<td>Enzyme Commission (EC) number terms</td>
</tr>
<tr>
<td><a href="EmbeddingsV1/">EmbeddingsV1</a></td>
<td>Vector embeddings version 1 for samples and entities</td>
</tr>
<tr>
<td><a href="GoHierarchyFlat/">GoHierarchyFlat</a></td>
<td>Flattened GO hierarchy for efficient ancestor/descendant queries</td>
</tr>
<tr>
<td><a href="GoTerms/">GoTerms</a></td>
<td>Gene Ontology terms with full metadata</td>
</tr>
<tr>
<td><a href="GottchaGold/">GottchaGold</a></td>
<td>GOTTCHA taxonomic classifications for GOLD samples</td>
</tr>
<tr>
<td><a href="KeggKoTerms/">KeggKoTerms</a></td>
<td>KEGG Orthology (KO) terms</td>
</tr>
<tr>
<td><a href="KeggPathwayTerms/">KeggPathwayTerms</a></td>
<td>KEGG pathway definitions with category classification</td>
</tr>
<tr>
<td><a href="KrakenGold/">KrakenGold</a></td>
<td>Kraken taxonomic classifications for GOLD samples</td>
</tr>
<tr>
<td><a href="LipidomicsGold/">LipidomicsGold</a></td>
<td>Lipidomics data linked to GOLD samples</td>
</tr>
<tr>
<td><a href="MetabolomicsGold/">MetabolomicsGold</a></td>
<td>Metabolomics data linked to GOLD samples</td>
</tr>
<tr>
<td><a href="MetacycPathwayReactions/">MetacycPathwayReactions</a></td>
<td>Reactions within MetaCyc pathways</td>
</tr>
<tr>
<td><a href="MetacycPathways/">MetacycPathways</a></td>
<td>MetaCyc metabolic pathways with hierarchical classification</td>
</tr>
<tr>
<td><a href="MetatranscriptomicsGold/">MetatranscriptomicsGold</a></td>
<td>Metatranscriptomics expression data linked to GOLD samples</td>
</tr>
<tr>
<td><a href="NomFeatureMetadata/">NomFeatureMetadata</a></td>
<td>NOM feature metadata including molecular formulas, exact masses, and compound...</td>
</tr>
<tr>
<td><a href="NomGold/">NomGold</a></td>
<td>Natural organic matter data linked to GOLD samples</td>
</tr>
<tr>
<td><a href="NomMatrixOptimized/">NomMatrixOptimized</a></td>
<td>Optimized NOM feature matrix for efficient queries</td>
</tr>
<tr>
<td><a href="OmicsFilesTable/">OmicsFilesTable</a></td>
<td>Inventory of omics data files with metadata and URLs</td>
</tr>
<tr>
<td><a href="ProteomicsGold/">ProteomicsGold</a></td>
<td>Proteomics data linked to GOLD samples</td>
</tr>
<tr>
<td><a href="RheaCrossrefs/">RheaCrossrefs</a></td>
<td>Cross-references from Rhea reactions to other databases</td>
</tr>
<tr>
<td><a href="RheaReactions/">RheaReactions</a></td>
<td>Rhea biochemical reactions database</td>
</tr>
<tr>
<td><a href="SampleFileLookup/">SampleFileLookup</a></td>
<td>Sample to file mapping for data retrieval</td>
</tr>
<tr>
<td><a href="SampleFileSelections/">SampleFileSelections</a></td>
<td>User-curated file selections per sample for analysis</td>
</tr>
<tr>
<td><a href="SampleTokensV1/">SampleTokensV1</a></td>
<td>Sample-level token assignments from vocabulary</td>
</tr>
<tr>
<td><a href="StudyTable/">StudyTable</a></td>
<td>NMDC research studies with ecosystem classification and investigator informat...</td>
</tr>
<tr>
<td><a href="TaxonomyDim/">TaxonomyDim</a></td>
<td>Taxonomic hierarchy dimension table using NCBI taxonomy</td>
</tr>
<tr>
<td><a href="TaxonomyEmbeddings/">TaxonomyEmbeddings</a></td>
<td>Taxonomic profile embeddings - vector representation of community composition...</td>
</tr>
<tr>
<td><a href="TaxonomyFeatures/">TaxonomyFeatures</a></td>
<td>Taxonomy-derived features for machine learning</td>
</tr>
<tr>
<td><a href="TraitEmbeddings/">TraitEmbeddings</a></td>
<td>Trait-based embeddings derived from functional annotations</td>
</tr>
<tr>
<td><a href="TraitFeatures/">TraitFeatures</a></td>
<td>Trait-derived features for machine learning</td>
</tr>
<tr>
<td><a href="TraitSources/">TraitSources</a></td>
<td>Sources for trait data (databases, literature, predictions)</td>
</tr>
<tr>
<td><a href="TraitTaxonomyMapping/">TraitTaxonomyMapping</a></td>
<td>Mapping between traits and taxonomic groups</td>
</tr>
<tr>
<td><a href="TraitUnified/">TraitUnified</a></td>
<td>Unified trait annotations across samples from multiple sources</td>
</tr>
<tr>
<td><a href="UnifiedEmbeddings/">UnifiedEmbeddings</a></td>
<td>Unified multi-modal embeddings combining taxonomy, traits, abiotic factors, a...</td>
</tr>
<tr>
<td><a href="VocabRegistryV1/">VocabRegistryV1</a></td>
<td>Vocabulary registry for multi-modal tokenization</td>
</tr>
</tbody>
</table>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Slot</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="__index_level_0__/"><strong>index_level_0</strong></a></td>
<td>Sample identifier (biosample ID)</td>
</tr>
<tr>
<td><a href="all_ancestors/">all_ancestors</a></td>
<td>Semicolon-separated all ancestor GO IDs (transitive closure)</td>
</tr>
<tr>
<td><a href="all_parents/">all_parents</a></td>
<td>Semicolon-separated direct parent GO IDs (immediate is_a/part_of parents)</td>
</tr>
<tr>
<td><a href="associated_dois/">associated_dois</a></td>
<td>JSON array of DOI objects with doi_value, doi_category (dataset_doi, award_do...</td>
</tr>
<tr>
<td><a href="category/">category</a></td>
<td>KEGG functional category</td>
</tr>
<tr>
<td><a href="category_code/">category_code</a></td>
<td>Single-letter COG category code (A-Z)</td>
</tr>
<tr>
<td><a href="category_name/">category_name</a></td>
<td>Full category name</td>
</tr>
<tr>
<td><a href="chebi/">chebi</a></td>
<td>ChEBI compound ID</td>
</tr>
<tr>
<td><a href="class/">class</a></td>
<td>Class name</td>
</tr>
<tr>
<td><a href="cog_id/">cog_id</a></td>
<td>Internal COG category ID (same as category_code)</td>
</tr>
<tr>
<td><a href="color_code/">color_code</a></td>
<td>Hex color code for visualization (without # prefix)</td>
</tr>
<tr>
<td><a href="confidence_level/">confidence_level</a></td>
<td>Confidence assessment of annotations from this source</td>
</tr>
<tr>
<td><a href="confidence_score/">confidence_score</a></td>
<td>Confidence score for this mapping (0-1)</td>
</tr>
<tr>
<td><a href="coverage_count/">coverage_count</a></td>
<td>Number of trait-taxon assignments from this source</td>
</tr>
<tr>
<td><a href="definition/">definition</a></td>
<td>Full term definition with citations in double quotes</td>
</tr>
<tr>
<td><a href="depth/">depth</a></td>
<td>Maximum depth from root (root terms have depth 1)</td>
</tr>
<tr>
<td><a href="description/">description</a></td>
<td>Term description or definition</td>
</tr>
<tr>
<td><a href="dim_0/">dim_0</a></td>
<td>First embedding dimension</td>
</tr>
<tr>
<td><a href="dim_255/">dim_255</a></td>
<td>Last embedding dimension (256 total dimensions, 0-255)</td>
</tr>
<tr>
<td><a href="ec_id/">ec_id</a></td>
<td>EC number in X</td>
</tr>
<tr>
<td><a href="ecosystem/">ecosystem</a></td>
<td>Top-level ecosystem classification</td>
</tr>
<tr>
<td><a href="ecosystem_category/">ecosystem_category</a></td>
<td>Ecosystem category (second level)</td>
</tr>
<tr>
<td><a href="ecosystem_subtype/">ecosystem_subtype</a></td>
<td>Ecosystem subtype for further classification</td>
</tr>
<tr>
<td><a href="ecosystem_type/">ecosystem_type</a></td>
<td>Specific ecosystem type (third level)</td>
</tr>
<tr>
<td><a href="entity_key/">entity_key</a></td>
<td>Unique key for this entity within its type</td>
</tr>
<tr>
<td><a href="entity_type/">entity_type</a></td>
<td>Type of entity this token represents</td>
</tr>
<tr>
<td><a href="evidence_type/">evidence_type</a></td>
<td>Type of evidence supporting this mapping</td>
</tr>
<tr>
<td><a href="family/">family</a></td>
<td>Family name</td>
</tr>
<tr>
<td><a href="feature_id/">feature_id</a></td>
<td>Metabolite feature ID (unique within file)</td>
</tr>
<tr>
<td><a href="file_id/">file_id</a></td>
<td>NMDC data object ID for source file</td>
</tr>
<tr>
<td><a href="file_name/">file_name</a></td>
<td>Original CSV file name</td>
</tr>
<tr>
<td><a href="funding_sources/">funding_sources</a></td>
<td>JSON array of funding sources</td>
</tr>
<tr>
<td><a href="genus/">genus</a></td>
<td>Genus name</td>
</tr>
<tr>
<td><a href="go_id/">go_id</a></td>
<td>GO term ID in GO:NNNNNNN format</td>
</tr>
<tr>
<td><a href="gold_study_identifiers/">gold_study_identifiers</a></td>
<td>JSON array of GOLD study IDs</td>
</tr>
<tr>
<td><a href="human_name/">human_name</a></td>
<td>Human-readable name for the token</td>
</tr>
<tr>
<td><a href="inchi/">inchi</a></td>
<td>InChI (International Chemical Identifier) string</td>
</tr>
<tr>
<td><a href="inchikey/">inchikey</a></td>
<td>InChIKey - 27-character hash of InChI for database searching</td>
</tr>
<tr>
<td><a href="Intensity/">Intensity</a></td>
<td>Signal intensity (peak height)</td>
</tr>
<tr>
<td><a href="is_obsolete/">is_obsolete</a></td>
<td>Whether term is deprecated and should not be used for new annotations</td>
</tr>
<tr>
<td><a href="kegg/">kegg</a></td>
<td>KEGG compound ID (C##### format)</td>
</tr>
<tr>
<td><a href="kingdom/">kingdom</a></td>
<td>Kingdom/superkingdom name</td>
</tr>
<tr>
<td><a href="ko_id/">ko_id</a></td>
<td>KEGG Orthology ID in KXXXXX format</td>
</tr>
<tr>
<td><a href="mapping_id/">mapping_id</a></td>
<td>Unique identifier for this trait-taxon mapping</td>
</tr>
<tr>
<td><a href="modality_id/">modality_id</a></td>
<td>Data modality (taxonomy, trait, abiotic, biochemical)</td>
</tr>
<tr>
<td><a href="Molecular_Formula/">Molecular_Formula</a></td>
<td>Chemical formula when determined from isotope patterns</td>
</tr>
<tr>
<td><a href="mz/">mz</a></td>
<td>Mass-to-charge ratio (m/z)</td>
</tr>
<tr>
<td><a href="name/">name</a></td>
<td>Human-readable term name/label</td>
</tr>
<tr>
<td><a href="namespace/">namespace</a></td>
<td>Ontology namespace (primarily for GO terms)</td>
</tr>
<tr>
<td><a href="order/">order</a></td>
<td>Order name</td>
</tr>
<tr>
<td><a href="organism_name/">organism_name</a></td>
<td>Organism name from source database</td>
</tr>
<tr>
<td><a href="parent_pathway/">parent_pathway</a></td>
<td>Parent pathway in hierarchy</td>
</tr>
<tr>
<td><a href="pathway_id/">pathway_id</a></td>
<td>KEGG pathway ID (ko or map prefix)</td>
</tr>
<tr>
<td><a href="phylum/">phylum</a></td>
<td>Phylum name</td>
</tr>
<tr>
<td><a href="principal_investigator_name/">principal_investigator_name</a></td>
<td>PI name</td>
</tr>
<tr>
<td><a href="principal_investigator_orcid/">principal_investigator_orcid</a></td>
<td>PI ORCID identifier for disambiguation</td>
</tr>
<tr>
<td><a href="Retention_Time_min/">Retention_Time_min</a></td>
<td>Chromatographic retention time in minutes</td>
</tr>
<tr>
<td><a href="RHEA_ID_BI/">RHEA_ID_BI</a></td>
<td>Bidirectional reaction ID</td>
</tr>
<tr>
<td><a href="RHEA_ID_LR/">RHEA_ID_LR</a></td>
<td>Left-to-right reaction ID</td>
</tr>
<tr>
<td><a href="RHEA_ID_MASTER/">RHEA_ID_MASTER</a></td>
<td>Master reaction ID</td>
</tr>
<tr>
<td><a href="RHEA_ID_RL/">RHEA_ID_RL</a></td>
<td>Right-to-left reaction ID</td>
</tr>
<tr>
<td><a href="rule_definition/">rule_definition</a></td>
<td>For rule-based sources, the logical rule definition</td>
</tr>
<tr>
<td><a href="sample_id/">sample_id</a></td>
<td>Sample identifier</td>
</tr>
<tr>
<td><a href="smiles/">smiles</a></td>
<td>SMILES notation for chemical structure</td>
</tr>
<tr>
<td><a href="source/">source</a></td>
<td>Source ontology/database for this term</td>
</tr>
<tr>
<td><a href="source_database/">source_database</a></td>
<td>Database of origin</td>
</tr>
<tr>
<td><a href="source_id/">source_id</a></td>
<td>Unique identifier for the trait source</td>
</tr>
<tr>
<td><a href="source_modality/">source_modality</a></td>
<td>Data modality this token comes from</td>
</tr>
<tr>
<td><a href="source_name/">source_name</a></td>
<td>Human-readable source name</td>
</tr>
<tr>
<td><a href="source_type/">source_type</a></td>
<td>Type of source (curated, rule_based, literature, computed)</td>
</tr>
<tr>
<td><a href="species/">species</a></td>
<td>Species name (binomial or with identifier)</td>
</tr>
<tr>
<td><a href="specific_ecosystem/">specific_ecosystem</a></td>
<td>Most specific ecosystem classification</td>
</tr>
<tr>
<td><a href="study_category/">study_category</a></td>
<td>Category of study</td>
</tr>
<tr>
<td><a href="study_id/">study_id</a></td>
<td>NMDC study identifier</td>
</tr>
<tr>
<td><a href="synonyms/">synonyms</a></td>
<td>Semicolon-separated list of term synonyms</td>
</tr>
<tr>
<td><a href="taxid/">taxid</a></td>
<td>NCBI taxonomy ID (integer)</td>
</tr>
<tr>
<td><a href="taxon_id/">taxon_id</a></td>
<td>NCBI taxonomy ID or IMG taxon_oid</td>
</tr>
<tr>
<td><a href="term_id/">term_id</a></td>
<td>Term identifier with format varying by source</td>
</tr>
<tr>
<td><a href="title/">title</a></td>
<td>Formal study title (may differ from name)</td>
</tr>
<tr>
<td><a href="token_id/">token_id</a></td>
<td>Unique token ID in vocabulary</td>
</tr>
<tr>
<td><a href="trait_category/">trait_category</a></td>
<td>Category of trait (phenotype, metabolism, energy_source, oxygen_req, cell_sha...</td>
</tr>
<tr>
<td><a href="trait_id/">trait_id</a></td>
<td>Unified trait identifier</td>
</tr>
<tr>
<td><a href="trait_name/">trait_name</a></td>
<td>Human-readable trait name</td>
</tr>
<tr>
<td><a href="value/">value</a></td>
<td>Token value/weight (e</td>
</tr>
<tr>
<td><a href="websites/">websites</a></td>
<td>JSON array of associated website URLs</td>
</tr>
</tbody>
</table>
<h2 id="enumerations">Enumerations</h2>
<table>
<thead>
<tr>
<th>Enumeration</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="AnnotationSource/">AnnotationSource</a></td>
<td>Source databases for annotation terms</td>
</tr>
<tr>
<td><a href="CogFunctionalCategory/">CogFunctionalCategory</a></td>
<td>COG single-letter functional categories (A-Z)</td>
</tr>
<tr>
<td><a href="Ecosystem.md">Ecosystem</a></td>
<td>Top-level ecosystem classification from GOLD</td>
</tr>
<tr>
<td><a href="EcosystemCategory/">EcosystemCategory</a></td>
<td>Ecosystem category within top-level ecosystem classification</td>
</tr>
<tr>
<td><a href="EcosystemType/">EcosystemType</a></td>
<td>Specific ecosystem types within categories</td>
</tr>
<tr>
<td><a href="GoNamespace/">GoNamespace</a></td>
<td>Gene Ontology namespaces (aspects)</td>
</tr>
<tr>
<td><a href="SourceModality/">SourceModality</a></td>
<td>Data modality from which tokens/embeddings are derived</td>
</tr>
<tr>
<td><a href="StudyCategory/">StudyCategory</a></td>
<td>Study organization type - individual research or consortium</td>
</tr>
<tr>
<td><a href="TokenEntityType/">TokenEntityType</a></td>
<td>Types of entities in the vocabulary registry for tokenization</td>
</tr>
</tbody>
</table>
<h2 id="types">Types</h2>
<table>
<thead>
<tr>
<th>Type</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Boolean/">Boolean</a></td>
<td>A binary (true or false) value</td>
</tr>
<tr>
<td><a href="Curie/">Curie</a></td>
<td>a compact URI</td>
</tr>
<tr>
<td><a href="Date/">Date</a></td>
<td>a date (year, month and day) in an idealized calendar</td>
</tr>
<tr>
<td><a href="DateOrDatetime/">DateOrDatetime</a></td>
<td>Either a date or a datetime</td>
</tr>
<tr>
<td><a href="Datetime/">Datetime</a></td>
<td>The combination of a date and time</td>
</tr>
<tr>
<td><a href="Decimal/">Decimal</a></td>
<td>A real number with arbitrary precision that conforms to the xsd:decimal speci...</td>
</tr>
<tr>
<td><a href="Double/">Double</a></td>
<td>A real number that conforms to the xsd:double specification</td>
</tr>
<tr>
<td><a href="Float/">Float</a></td>
<td>A real number that conforms to the xsd:float specification</td>
</tr>
<tr>
<td><a href="Integer/">Integer</a></td>
<td>An integer</td>
</tr>
<tr>
<td><a href="Jsonpath/">Jsonpath</a></td>
<td>A string encoding a JSON Path</td>
</tr>
<tr>
<td><a href="Jsonpointer/">Jsonpointer</a></td>
<td>A string encoding a JSON Pointer</td>
</tr>
<tr>
<td><a href="Ncname/">Ncname</a></td>
<td>Prefix part of CURIE</td>
</tr>
<tr>
<td><a href="Nodeidentifier/">Nodeidentifier</a></td>
<td>A URI, CURIE or BNODE that represents a node in a model</td>
</tr>
<tr>
<td><a href="Objectidentifier/">Objectidentifier</a></td>
<td>A URI or CURIE that represents an object in the model</td>
</tr>
<tr>
<td><a href="Sparqlpath/">Sparqlpath</a></td>
<td>A string encoding a SPARQL Property Path</td>
</tr>
<tr>
<td><a href="String/">String</a></td>
<td>A character string</td>
</tr>
<tr>
<td><a href="Time/">Time</a></td>
<td>A time object represents a (local) time of day, independent of any particular...</td>
</tr>
<tr>
<td><a href="Uri/">Uri</a></td>
<td>a complete URI</td>
</tr>
<tr>
<td><a href="Uriorcurie/">Uriorcurie</a></td>
<td>a URI or a CURIE</td>
</tr>
</tbody>
</table>
<h2 id="subsets">Subsets</h2>
<table>
<thead>
<tr>
<th>Subset</th>
<th>Description</th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
<td></td>
</tr>
</tbody>
</table>












                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": "../..", "features": [], "search": "../../assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.79ae519e.min.js"></script>
      
    
  </body>
</html>